webapp/utils/async_client.jsx

```diff
@@ -808,6 +808,29 @@ export function savePreferences(preferences, success, error) {
     );
 }
 
+export function deletePreferences(preferences, success, error) {
```

Comment:
I wanted to stay consistent with other calls where there's an async version and a regular one. I imagine that there's other pieces of code that might want to call this in the future without talking to the client directly.